lib/pkg/flow-base/lib/FlowHelpers.js

```diff
@@ -144,12 +144,28 @@ async function canFindFlow(flowPath: string): Promise<boolean> {
 }
 
 /**
- * @return The path to Flow on the user's machine. It is recommended not to cache the result of this
- *   function in case the user updates his or her preferences in Atom, in which case the return
- *   value will be stale.
+ * @return The path to Flow on the user's machine. First using the the user's config, then looking into
+ *   the node_modules for the project.
+ *   It is recommended not to cache the result of this function in case the user updates his or her 
+ *   preferences in Atom, in which case the return value will be stale.
  */
-function getPathToFlow(): string {
-  return global.vscode.workspace.getConfiguration('flow').get('pathToFlow')
+async function getPathToFlow(): Promise<string> {
+  const config = global.vscode.workspace.getConfiguration('flow').get('pathToFlow')
+  if (await canFindFlow(config)) {
+    return config
+  }
+  if (await canFindFlow(fallbackNodeModuleFlowLocation())){
+    return fallbackNodeModuleFlowLocation()
+  }
+  return config
+}
+
+/**
+ * @return The potential path to Flow on the user's machine if they are using NPM/Yarn to manage
+ * their installs of flow.
+ */
+function fallbackNodeModuleFlowLocation(): string {
+  return global.vscode.workspace.rootPath + "/node_modules/.bin/flow"
```

Comment:
Have you tested this on Windows? I think this would work, but not near a Windows computer so can't double check. I vaguely remember that there's a `node_modules/.bin/flow.cmd` which is what you can use from the commandline, but I wonder if there is a `.bin/flow` or if the `flow.cmd` just references a binary inside of `node_modules/flow`